migration/src/test/java/uk/ac/bbsrc/tgac/miso/migration/destination/ValueTypeLookupTest.java

```diff
@@ -289,72 +290,80 @@ private KitDescriptor makeKit(Long id, String name) {
   @Test
   public void testResolveSamplePurpose() {
     assertNotNull(sut.resolve(makeSamplePurpose(VALID_LONG, null)));
-    assertNotNull(sut.resolve(makeSamplePurpose(SamplePurposeImpl.UNSAVED_ID, VALID_STRING)));
+    assertNotNull(sut.resolve(makeSamplePurpose(null, VALID_STRING)));
     assertNull(sut.resolve((SamplePurpose) null));
-    assertNull(sut.resolve(makeSamplePurpose(SamplePurposeImpl.UNSAVED_ID, null)));
+    assertNull(sut.resolve(makeSamplePurpose(null, null)));
     assertNull(sut.resolve(makeSamplePurpose(INVALID_LONG, null)));
-    assertNull(sut.resolve(makeSamplePurpose(SamplePurposeImpl.UNSAVED_ID, INVALID_STRING)));
+    assertNull(sut.resolve(makeSamplePurpose(null, INVALID_STRING)));
   }
 
-  private SamplePurpose makeSamplePurpose(long id, String alias) {
+  private SamplePurpose makeSamplePurpose(Long id, String alias) {
     SamplePurpose sp = new SamplePurposeImpl();
-    sp.setId(id);
+    if (id != null) {
+      sp.setId(id);
+    }
     sp.setAlias(alias);
     return sp;
   }
 
   @Test
   public void testResolveLab() {
-    assertNotNull(sut.resolve(makeLab(VALID_LONG, null, InstituteImpl.UNSAVED_ID, null)));
-    assertNotNull(sut.resolve(makeLab(LabImpl.UNSAVED_ID, VALID_STRING, VALID_LONG, null)));
-    assertNotNull(sut.resolve(makeLab(LabImpl.UNSAVED_ID, VALID_STRING, InstituteImpl.UNSAVED_ID, VALID_STRING)));
+    // assertNotNull(sut.resolve(makeLab(VALID_LONG, null, null, null)));
```

Comment:
Should this test just be deleted?

Reply:
Nope, I didn't mean to leave these commented out. Fixed now